docker/wiremock/__files/swanseaOrg.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "userIdentifier": "acfe91d3-745f-4e60-8272-2cada36152a5",
+    "firstName": "damian@swansea.gov.uk",
+    "lastName": "(local-authority)",
+    "email": "damian@swansea.gov.uk",
+    "roles": [
+      "caseworker",
+      "caseworker-publiclaw",
+      "caseworker-publiclaw-solicitor"
+    ],
+    "idamStatus": "ACTIVE?",
```

Comment:
Out of curiosity what does these questions marks mean?

Reply:
That I was not sure at the time of writing what the endpoint returns (as the API is string and not enum...)

Reply:
Removed the question marks.